game/Content/FleetBuildRatiosRandomization.yaml

```diff
@@ -0,0 +1,83 @@
+# This will set the Fleet Randomized Ratios for the AI ship build logic. 
```

Comment:
This file is not needed.
In the fleetRatios.yaml, you can use Range struct instead of duplicating a yaml file.
check Range.cs. and its implementation in PlanetTypes.yaml (PopPerTile, for instance)
The Range struct has the Generate method which will generate a random number for the provided range. 
It can also get only the minimum or the maximum numbers, incase a modder does not want random numbers.

Reply:
done